### PR TITLE
API endpoint to get all public events

### PR DESCRIPTION
**Pull request by Lameya56**

### CODE REVIEW

File: handlers/events.go [hunk 1]
```diff
@@ -64,3 +64,17 @@ func GetEvent(c *gin.Context) {
...
+    setCors(c)
+
+    // Fetch all public events from the database
+    publicEvents, err := eventsdb.GetAllPublicEvents()
+    if err != nil {
+        fmt.Printf("ERROR: %+v", err)
+        c.IndentedJSON(http.StatusInternalServerError, nil) //server error
+        return
+    }
+	fmt.Printf("public events")
```
Comment (dianaabishop) [inline handlers/events.go:78]: lil cleanup thing - I think you can remove this print statement!

File: main.go [hunk 2]
```diff
@@ -35,6 +35,9 @@ func main() {
 	api.GET("/event/:eventID", handlers.GetEvent)
 	api.OPTIONS("/event/:eventID", handlers.HandleCors)
 
+	//GET PUBLIC EVENTS
+	api.GET("/public-events", handlers.GetPublicEvents)
+
```
Comment (dianaabishop) [inline main.go:40]: you might need a corresponding CORS endpoint like the endpoint above for the FE to be able to call this (or you might not, I forget if they're needed for GET requests 🤔 or if we only ran into the issue with POST/PATCH/PUT/DELETE).

File: views/js/evently/src/AboutUs.css [hunk 3]
```diff
@@ -1,4 +1,40 @@
 .AboutUs {
     text-align: center;
-   line-height: 30rem;
-}
\ No newline at end of file
+}
```
Comment (dianaabishop) [inline views/js/evently/src/AboutUs.css:3]: I think these FE changes (and in the files elow) should be on your other branch? Let me know if you need help transferring it to your About Us branch/PR!

**Review by dianaabishop — COMMENTED** [review]
Looking great!! Just a few cleanup changes and this should be good to go 🚀

File: views/js/evently/src/AboutUs.css (hunk 3 above)
Comment (Lameya56) [reply views/js/evently/src/AboutUs.css:3]: I think I only added the backend changes and made sure I was on the get_all_events branch so I am not sure if the FE work got pushed as well.

Comment (dianaabishop) [reply views/js/evently/src/AboutUs.css:3]: It appears to have! it's pretty easy to accidentally push work to the wrong branch; it's no big deal! we should definitely move it to your other branch before this is tested/merged into main though - let me know if you need help doing that!

**Review by dianaabishop — APPROVED** [review]
LGTM! Ready for merge to main!

**Pull request merged**